examples/official-site/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
+Disallow: /examples/hash_password.sql  # Block access to specific sensitive SQL files like this
```

Comment:
Why ? This is a useful page, it is not password protected: https://sql.datapage.app/examples/hash_password.sql

Reply:
which one is ?